crap-advisor-frontend/src/core/components/Sidebar/index.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import "./index.css"
+import logo from "../../../assets/logo.png"
 
 const Sidebar = () => {
     return (
         <div className={"sidebar"}>
-            <h3 id={"appTitle"}>Crap Advisor</h3>
+            <img className={"logo"} src={logo} alt={"Crap Advisor"}/>
```

Comment:
Do not write className prop in curly brakes

Reply:
gotcha